### PR TITLE
Rearrange for pip install

### PR DESCRIPTION
**Pull request by damkjaer**
This is a rearrangement of the source for being able to install mesos-tools using pip:
  pip install git+https://github.com/DBCDK/mesos-tools

There has also been added a travis file, which should be functional.

Test has been mangled a little, and it may need for developers to add src/mesos-tools to their PYTHONPATH before testing (using pytest)

### CODE REVIEW

**Review by ApoDBC — APPROVED** [review]

**Review by jbndbc — APPROVED** [review]

**Review by thldbc — APPROVED** [review]
Ok

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```
Comment (jensstn) [inline setup.py:2]: det her er ikke nødvendigt i python3

File: tests/test_marathon_config_producer.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 import unittest
 
-import marathon_config_producer
+from mesos_tools import marathon_config_producer
```
Comment (jensstn) [inline tests/test_marathon_config_producer.py:7]: irriterende at man ikke kan køre testne med ```python3 -m unittest discover``` længere

**Review by jensstn — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (jensstn) [reply setup.py:2]: hov - jeg mente kun at ```coding: utf-8``` ikke er nødvendigt

**Pull request merged**